monitoring/osd-managing-metrics.adoc

```diff
@@ -0,0 +1,34 @@
+[id="managing-metrics"]
+= Managing metrics
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: managing-metrics
+
+toc::[]
+
+This section provides an overview about how {product-title} metrics are collected, queried and visualized.
```

Comment:
For assembly introductions, you could change "section" to "document", since the reference is to the whole assembly.

Reply:
Done.